applications/crossbar/priv/couchdb/account/access_lists.json

```diff
@@ -0,0 +1,9 @@
+{
+    "_id": "_design/access_lists"
+    ,"language": "javascript"
+    ,"views": {
+        "crossbar_listing": {
+            "map": "function(doc) { if ((doc.pvt_type != 'account' && doc.pvt_type != 'device') || doc.pvt_deleted || !doc.access_lists) return; var name = doc.realm || doc.sip.username; var type = ''; if (doc.pvt_type == 'account') type = 'realm'; else type = 'device'; emit(name, { 'type': type, 'acls': doc.access_lists }); };"
```

Comment:
I wouldn't bother limiting `doc.pvt_type` to account and device types. Just check for pvt_deleted or no access_lists. It could be, in the future, that other doc types will want access_lists added.